### ISSUE TITLE
[Adaptive Card iOS - ActionModeTestCard.json]: Voice over is conveying incorrect name for '...' control.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8400]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice Over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Test' control and activate it.
6. Navigate to 'IActionModeTestCard.json' under 'Test' list item and activate it.
7.  Card will be appear.
8. Navigate to '...' control
9. Observe 'voice over' issue.

**Actual Result:**
Voice over is conveying incorrect name for '...' button. It is announcing as **'comma ellipse comma'** for '...' control.

**Expected Result:**
Voice over should convey the correct name for the '...' button and it should announce as **'More Option'** button.

**Note:**
This issue is repro throughout the screen for the same control.

**User Impact:**
It will impact the screen reader user if the screen reader does convey the incorrect name for any controls on the screen as they will not be able to know about the actual use of the button.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/227786474-52025e3e-a4da-4609-ab79-4f92a7d8da54.mov

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
closing this issue, the app wasn't built correctly perhaps, only cards under Scenario should be tested, not under Elements. I will make note of this and fix it.

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as per the above comment.
